packages/ui/src/forum/types/ForumCategory.ts

```diff
@@ -16,3 +20,33 @@ export const asForumCategory = (fields: ForumCategoryFieldsFragment): ForumCateg
   description: fields.description,
   subcategories: [],
 })
+
+const asSubCategory = (fields: ForumSubCategoryFieldsFragment): ForumSubCategory => ({
```

Comment:
Let's call it `ForumCategoryBreadcrumb` and `asForumCategoryBreadcrumb()` (I think that dropping `Category` will be also OK: `ForumBreadcrumb` should be understandable and will directly tie to the usage. The problem with `ForumSubCategory` for me is that a sub-category is every category nested inside another and is tied (at least for me) to the whole: category, threads, own page, etc.